tensorflow/lite/micro/integration_tests/generate_per_layer_tests.py

```diff
@@ -168,6 +168,8 @@ def generate_goldens(self, builtin_operator):
 
       if builtin_operator == schema_fb.BuiltinOperator.CONV_2D:
         generated_inputs = self.generate_inputs_conv(interpreter)
+      if builtin_operator == schema_fb.BuiltinOperator.SVDF:
+        generated_inputs = self.generate_inputs_conv(interpreter)
```

Comment:
we will want to rename the generate_inputs_conv function to something more generic since it is useful for both conv and svdf.

I'm happy to make such a change.

Reply:
@advaitjain I will raise new PR with more ops added together with a proper naming convention